client/chatline.cpp

```diff
@@ -385,7 +577,7 @@ void chat_widget::set_chat_visible(bool visible)
 }
 
 /**
- * Shows the chat and ensures the chat line has focus
+   Shows the chat and ensures the chat line has focus
```

Comment:
The * was correct :D
```suggestion
 * Shows the chat and ensures the chat line has focus
```

Reply:
Interesting. How so? It was the only header like it in the file

Reply:
https://longturn.readthedocs.io/en/latest/Coding/guidelines.html#documentation-comments

Reply:
Not consistent then. most of the function comments in the file use this convention

```
/**
   some comment
*/
```

Reply:
Yes, most functions are still like this because I didn't dare mass-replacing the old style.